infra/config-store/src/main/java/com/simprints/infra/config/store/ConfigRepositoryImpl.kt

```diff
@@ -81,6 +91,28 @@ internal class ConfigRepositoryImpl @Inject constructor(
         }
     }
 
+    private suspend fun tokenizeModules(config: ProjectConfiguration): ProjectConfiguration {
```

Comment:
Is this really needed? [Looks like](https://github.com/Simprints/Android-Simprints-ID/blob/1b8c10c6e84da68ee41f30ee2b5c10c4710ff222/infra/config-store/src/main/java/com/simprints/infra/config/store/remote/models/ApiSynchronizationConfiguration.kt#L106) we get the module options already tokenized from the API. The only option I can see is for the device to have just upgraded from a pre-tokenization version and not yet synced new configuration. But in this case it would not have the keys to encrypt modules, anyway.